docs/using.adoc

```diff
@@ -314,3 +314,15 @@ Destroys the Vagrant box permanently.
 WARNING: Using `vagrant destroy` will destroy any data you stored in
 the Vagrant box. You will not be able to restart this instance and will
 have to create a new one using `vagrant up`.
+
+`vagrant global status`::
+Displays the active Vagrant environments on your system. Use this to see the status of your Vagrant environment.
```

Comment:
Add and option here as `--prune`. You can check its doc here https://www.vagrantup.com/docs/cli/global-status.html